docs/content/architecture-buildtime.md

```diff
@@ -28,8 +28,8 @@ After the image is built, Porter then generates a CNAB-compliant bundle.json fil
 
 * [Create a bundle](/bundle/create/)
 * [Use a custom Dockerfile](/bundle/custom-dockerfile/)
-* [Mixin Architecture](/mixin-dev-guide/architecture/)
-* [Building the Invocation Image](/build-image/)
+* [Mixin Architecture](/how-to-guides/work-with-mixins)
+* [Building the Invocation Image](TODO)
```

Comment:
not sure what page this is referring to

Reply:
[Unsure, just like here I guess?](https://github.com/getporter/porter/blob/34ad0d1f95e6c53e27ddc20f77241692baac709a/docs/content/how-to-guides/work-with-mixins.md)

Reply:
I mean the second link

Reply:
Just remove it for now I guess 🤷🏻 
If ya can